lib/dpul_collections_web/live/home_live.ex

```diff
@@ -47,7 +47,6 @@ defmodule DpulCollectionsWeb.HomeLive do
                 <.genre_link filter="flyers" label={gettext("Flyers")} />
                 <.genre_link filter="leaflets" label={gettext("Leaflets")} />
                 <.genre_link filter="photographs" label={gettext("Photographs")} />
-                <.genre_link filter="art" label={gettext("Art")} />
```

Comment:
Replace with paintings? https://digital-collections.princeton.edu/search?filter[genre][]=Paintings&q=ARt&sort_by=relevance&page=1&per_page=50

Non-blocking, because I'm goin on vacation.